metrics/src/main/java/cucumber/metrics/regulator/module/SpeedRegulatorModule.java

```diff
@@ -0,0 +1,49 @@
+package cucumber.metrics.regulator.module;
```

Comment:
I was saying that `SpeedRegulatorModule.java` probably belongs to `guice` sub-project under `cucumber-jvm` as it is Guice-specific, instead of being included in `metrics`.  If one were to add support for metrics for Spring IOC, how would that look like?  Spring also support AOP interceptors, so the same interceptor can be wired by `SpringFactory` from `spring` sub-project to add `metrics` support as well.